src/Validator.ts

```diff
@@ -288,4 +292,18 @@ export class Validator implements IValidator {
     }
     return isValidRealm
   }
+
+  async updateTags (uuid: string, profile: AMTConfiguration): Promise<void> {
+    let tags = []
+    if (profile?.tags.length > 0) {
+      tags = profile.tags
+      await got(`${EnvReader.GlobalEnvConfig.mpsServer}/api/v1/devices`, {
+        method: 'POST',
```

Comment:
shouldn't this be PATCH? POST would upsert -- seems like it would clear out the rest of the data with null

Reply:
done